### PR TITLE
NMS-13267 PrometheusCollector docs fix

### PR DESCRIPTION
**Pull request by mmahacek**
Fix formatting and add suggested default URL value

### All Contributors

* [X] Have you read and followed our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [X] Have you [made an issue in the OpenNMS issue tracker](https://issues.opennms.org)?<br>If so, you should:
  1. update the title of this PR to be of the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
* [X] Have you made a comment in that issue which points back to this PR?
* [X] Have you updated the JIRA link at the bottom of this comment to link to your issue?
* [ ] If this is a new or updated feature, is there documentation for the new behavior?
* [ ] If this is new code, are there unit and/or integration tests?
* [ ] If this PR targets a `foundation-*` branch, does it avoid changing files in `$OPENNMS_HOME/etc/`?

### Pull Request Process

One or more reviewers should be assigned to each PR.

If you know that a particular person is subject matter expert in the area your PR affects, feel free to assign one or more reviewers when you create this PR, otherwise reviewers will be assigned for you.

If you have made additions or changes to the documentation, or if you _need_ documentation for these code changes, please make sure a technical writer has looked it over.

Once the reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the Order of the Green Polo) you are welcome to merge the PR.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): https://issues.opennms.org/browse/NMS-13267

### CODE REVIEW

File: docs/modules/operation/pages/performance-data-collection/collectors/prometheus.adoc [hunk 1]
```diff
@@ -23,10 +23,10 @@ This collector provides tools for parsing and mapping the metrics to the collect
 |===
 | Parameter       | Description                                    | Required | Default value
 | `collection`    | The name of the _Prometheus Collection_ to use | required |
-| `url`           | HTTP URL to query for the metrics              | required |
+| `url`           | HTTP URL to query for the metrics (most likely will be `http://INTERFACE_ADDRESS:9100/metrics`)  | required |
```
Comment (j-white) [inline docs/modules/operation/pages/performance-data-collection/collectors/prometheus.adoc:26]: Should we consider using the meta-data syntax i.e. `${interface:address}` here in the example instead `INTERFACE_ADDRESS`?

Comment (mmahacek) [reply docs/modules/operation/pages/performance-data-collection/collectors/prometheus.adoc:26]: given https://issues.opennms.org/browse/NMS-13257 I kept PSM docs consistent.

Comment (j-white) [reply docs/modules/operation/pages/performance-data-collection/collectors/prometheus.adoc:26]: Yeah, it doesn't work everywhere, but it does work more often than `INTERFACE_ADDRESS` does.

I'm good either way, just caught my attention :)

Comment (mmahacek) [reply docs/modules/operation/pages/performance-data-collection/collectors/prometheus.adoc:26]: Also: https://github.com/OpenNMS/opennms/blob/7e9f5759badaf5636ae487a72b605a2a89e3e13e/features/prometheus-collector/src/main/java/org/opennms/netmgt/collectd/prometheus/PrometheusCollector.java#L99

**Review by Bonrob2 — APPROVED** [review]
LGTM!

**Review by indigo423 — APPROVED** [review]
LGTM! 👍

File: docs/modules/operation/pages/performance-data-collection/collectors/prometheus.adoc (hunk 1 above)
Comment (mmahacek) [reply docs/modules/operation/pages/performance-data-collection/collectors/prometheus.adoc:26]: In a test, `${interface:address}` gives me a "bad substitution" error in Karaf while `INTERFACE_ADDRESS` returns data.  I'll open a JIRA as it doesn't look like meta data is properly supported on this collector.

Comment (dino2gnt) [reply docs/modules/operation/pages/performance-data-collection/collectors/prometheus.adoc:26]: you have to quote the value of the url parameter.  `url='http://${interface:address}:9182/metrics'` works.

Comment (mmahacek) [reply docs/modules/operation/pages/performance-data-collection/collectors/prometheus.adoc]: After realizing I had other errors in my Prometheus config, I was able to get collection using the meta-data syntax in my collectd service.  I haven't had a case where I've needed to collect from Prometheus, though I was curious if there is any reason to have a default value for the URL if that parameter is omitted. I don't know how common it is for people to override the port or `/metrics` path.

**Pull request merged**